src/app/public/modules/adapter-service/adapter.service.ts

```diff
@@ -191,6 +199,39 @@ export class SkyCoreAdapterService {
     return false;
   }
 
+  /**
+   * Remove inline height styles from the provided elements.
+   * @param elementRef - The element to search within.
+   * @param selector - The CSS selector to use when finding elements for removing height.
+   */
+  public resetHeight(elementRef: ElementRef, selector: string): void {
+    const children = elementRef.nativeElement.querySelectorAll(selector);
+    if (children.length > 0) {
+      for (let i = 0; i < children.length; i++) {
+        // tslint:disable-next-line: no-null-keyword
+        children[i].style.height = null;
+      }
+    }
+  }
+
+  /**
+   * Sets all element heights to match the height of the tallest element.
+   * @param elementRef - The element to search within.
+   * @param selector - The CSS selector to use when finding elements for syncing height.
+   */
+  public syncHeight(elementRef: ElementRef, selector: string): void {
```

Comment:
Thoughts on maybe naming this to `syncTallestHeight` or something similar in case we decide we need a method to sync the smallest height in the future?

Reply:
Good thought. I went with `syncMaxHeight`.